src/applications/static-pages/download-1095b/components/App/index.unit.spec.js

```diff
@@ -1,10 +1,105 @@
-// // Dependencies.
-// import React from 'react';
-// import { expect } from 'chai';
-// import { shallow, mount } from 'enzyme';
-// import { Provider } from 'react-redux';
-// // Relative imports.
-// import { App } from '.';
-
-// describe('Download 1095B <App>', () => {
-// });
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import configureStore from 'redux-mock-store';
+import { $ } from 'platform/forms-system/src/js/utilities/ui';
+import App from './index';
+
+const mockStore = configureStore([]);
+
+describe('App component', () => {
+  let store;
+
+  const unauthenticatedState = {
+    featureToggles: {
+      loading: false,
+      showDigitalForm1095b: true,
+    },
+    user: {
+      login: {
+        currentlyLoggedIn: false,
+      },
+      profile: {
+        loa: {
```

Comment:
Should the unauthenticated state have any loa? Seems like that wouldn't be known until the user signs in.

Reply:
Thanks for catching that. I've made an update to remove this bit of data, and how the app handles selecting it.